### PR TITLE
AMP-40: disable AMP for CMP

### PR DESCRIPTION
**Pull request by themech**

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [GoogleAmpHelper.class.php#L80](https://github.com/Wikia/app/blob/730d099b6e4e353d39ab66b42201dca3d960870e/extensions/wikia/GoogleAmp/GoogleAmpHelper.class.php#L80): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: extensions/wikia/GoogleAmp/GoogleAmpHelper.class.php [hunk 1]
```diff
@@ -13,7 +24,8 @@ static public function isAmpEnabled( Title $title ): bool {
 
 		if ( $title->exists() &&
 			in_array( $title->getNamespace(), $wgGoogleAmpNamespaces ) &&
-			!in_array( $title->getPrefixedDBkey(), $wgGoogleAmpArticleBlacklist ) ) {
+			!in_array( $title->getPrefixedDBkey(), $wgGoogleAmpArticleBlacklist ) &&
+			!self::isCuratedMainPage() ) {
```
Comment (rogatty) [inline extensions/wikia/GoogleAmp/GoogleAmpHelper.class.php]: Pass `$title` 😉

Comment (themech) [reply extensions/wikia/GoogleAmp/GoogleAmpHelper.class.php]: done, thanks

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [GoogleAmpHelper.class.php#L80](https://github.com/Wikia/app/blob/3ede08e9f368ebd79d32d9b2e62ec9c1c06a35a4/extensions/wikia/GoogleAmp/GoogleAmpHelper.class.php#L80): Remove the unused function parameter "$skin". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by rogatty — APPROVED** [review]

**Pull request merged**